### ISSUE TITLE
Add ability to listen for animation completion

### ISSUE THREAD

**Issue by slorber** [oblador/react-native-collapsible#202]
Hi,

I'd like to be able to listen for animation completion.

My usecase:
- I want a view to be collapsible
- When that is not collapsed, I want to make it scroll into view (like the DOM element function)
- I need to wait until animation completion before making it scroll into view to avoid unwanted behaviors
- The "scrollIntoView" implementation is provided by https://github.com/APSL/react-native-keyboard-aware-scroll-view

```js
import Collapsible from 'react-native-collapsible';
<Collapsible 
  collapsed={isCollapsed} 
  onAnimationEnd={isCollapsed => !isCollapsed && scrollIntoView(this.collapsedViewRef)}
>
  <CollapsedView ref={this.collapsedViewRef = ref}/>
</Collapsible>;
```

**Comment by iRoachie** [comment]
Already coming in the next release https://github.com/oblador/react-native-collapsible/commit/3a600ca6fe18a84c9dcdd358fe9b944d5b30f026

**Comment by slorber** [comment]
awesome thanks, will wait for that release

**Comment by iRoachie** [comment]
Released in [0.120](https://github.com/oblador/react-native-collapsible/releases/tag/v0.12.0)
